Source/santabundleservice/SNTBundleService.m

```diff
@@ -226,6 +226,17 @@ - (NSDictionary *)generateEventsFromBinaries:(NSArray *)fis
 
       MOLCodesignChecker *cs = [fi codesignCheckerWithError:NULL];
       se.signingChain = cs.certificates;
+      se.cdhash = cs.cdhash;
+      se.teamID = cs.teamID;
+      if (cs.signingID) {
+        if (cs.teamID) {
+          se.signingID = [NSString stringWithFormat:@"%@:%@", cs.teamID, cs.signingID];
+        } else {
+          if (cs.platformBinary) {
+            se.signingID = [NSString stringWithFormat:@"platform:%@", cs.signingID];
+          }
```

Comment:
```suggestion
        } else if (cs.platformBinary) {
          se.signingID = [NSString stringWithFormat:@"platform:%@", cs.signingID];
        }
```

Reply:
I have included this change in the last commit